### PR TITLE
Add width to ADMS source characteristics

### PR DESCRIPTION
**Pull request by Hilbrand**
width on road will be replaced by this, and removed in a follow up pr.

### CODE REVIEW

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/ADMSSourceCharacteristics.java [hunk 1]
```diff
@@ -121,6 +123,14 @@ public void setDiameter(final double diameter) {
     this.diameter = diameter;
   }
 
+  public final double getWidth() {
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/ADMSSourceCharacteristics.java:126]: final?

**Review by BertScholten — CHANGES_REQUESTED** [review]
Don't mind the change, but also don't see what this has to do with the width on `ADMSRoadEmissionSource`. That object doesn't use `ADMSSourceCharacteristics` does it?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/ADMSSourceCharacteristics.java (hunk 1 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/ADMSSourceCharacteristics.java]: hmm copied from characteristics.... will remove.

**Comment by Hilbrand** [comment]
You're right. width on road can remain: it doesn't use the characteristics class.

**Review by BertScholten — APPROVED** [review]
LGTM

**Pull request merged**